src/ducks/transactions/Transactions.styl

```diff
@@ -46,6 +46,9 @@
     .bnk-op-caticon
         cursor pointer
 
+    .bnk-op-clickable-area
+        cursor pointer
```

Comment:
I guess a utility class would be good :D

Reply:
:+1: :heavy_check_mark: